contracts/testing/TestUniswapV2Pair2.sol

```diff
@@ -0,0 +1,58 @@
+/*
+
+    Copyright 2020 dYdX Trading Inc.
+
+    Licensed under the Apache License, Version 2.0 (the "License");
+    you may not use this file except in compliance with the License.
+    You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+    Unless required by applicable law or agreed to in writing, software
+    distributed under the License is distributed on an "AS IS" BASIS,
+    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+    See the License for the specific language governing permissions and
+    limitations under the License.
+
+*/
+
+pragma solidity 0.5.7;
+pragma experimental ABIEncoderV2;
+
+import { IUniswapV2Pair } from "../external/interfaces/IUniswapV2Pair.sol";
+
+
+/**
+ * @title TestUniswapV2Pair2
+ * @author dYdX
+ *
+ * Another mock Uniswap V2 pair.
+ */
+contract TestUniswapV2Pair2 is
+    IUniswapV2Pair
```

Comment:
nit: you can just have it inherit from `TestUniswapV2Pair1` and add no new functions or variables